src/patternfly/components/Table/table-th.hbs

```diff
@@ -34,6 +34,14 @@
       aria-sort="none"
     {{/if}}
   {{/if}}
+
+  {{#ifAny table-th--style table-th--min-width}}
+    style="{{table-th--style}}{{#if table-th--min-width}} min-width: {{table-th--min-width}}{{/if}}"
+  {{/ifAny}}
+
+  {{#if table-th--attribute}}
+    {{{table-th--attribute}}}
+  {{/if}}
```

Comment:
looks like this is duped here? https://github.com/patternfly/patternfly/blob/75e61dbc2788655ecfda43ed90a6c7e8ec2c7d31/src/patternfly/components/Table/table-th.hbs#L51-L53